hw2/04-background.js

```diff
@@ -0,0 +1,37 @@
+// Add your code here
+const userInput = document.getElementById("interval");
+const button = document.querySelector("button");
+const body = document.querySelector("body");
+let toggle;
+
+function toggleBg() {
+  let interval = userInput.value;
+  console.log(`toggling interval with ${interval} seconds`);
+  if(!toggle) {
+    toggle = setInterval(setBackground, interval * 1000);
+    button.classList.remove("btn-primary");
+    button.classList.add("btn-danger");
```

Comment:
Kind of a nitpick, but you could condense these two lines into a single call to `button.className = 'btn btn-danger';` or whatever the equivalent list of classes might be. In other words, you could overwrite the list of classes all at once instead of adding or removing.